bigtable-hbase-integration-tests/pom.xml

```diff
@@ -104,8 +104,8 @@ limitations under the License.
                 </dependency>
                 <dependency>
                     <groupId>io.netty</groupId>
-                    <artifactId>netty-tcnative</artifactId>
-                    <version>1.1.33.Fork9</version>
+                    <artifactId>netty-tcnative-boringssl-static</artifactId>
+                    <version>1.1.33.Fork13</version>
                     <classifier>${os.detected.classifier}</classifier>
```

Comment:
Classifier is still needed.  There is an binary library in each of the jars that's built specifically for each OS.  The major difference between boringssl and openssl is the fact that we don't need an additional installation of the openssl binary; the OS specific jar is always needed.